internal/access/settings.go

```diff
@@ -0,0 +1,63 @@
+package access
+
+import (
+	"fmt"
+
+	"github.com/gin-gonic/gin"
+	"gopkg.in/square/go-jose.v2"
+
+	"github.com/infrahq/infra/api"
+	"github.com/infrahq/infra/internal/server/data"
+	"github.com/infrahq/infra/internal/server/models"
+)
+
+func GetPublicJWK(c *gin.Context) ([]jose.JSONWebKey, error) {
+	db := getDB(c)
+	settings, err := data.GetSettings(db)
+	if err != nil {
+		return nil, fmt.Errorf("could not get JWKs: %w", err)
+	}
+
+	var pubKey jose.JSONWebKey
+	if err := pubKey.UnmarshalJSON(settings.PublicJWK); err != nil {
+		return nil, fmt.Errorf("could not get JWKs: %w", err)
+	}
+
+	return []jose.JSONWebKey{pubKey}, nil
+}
+
+func GetPasswordRequirements(c *gin.Context) (api.PasswordRequirements, error) {
+	db := getDB(c)
+	settings, err := data.GetSettings(db)
+
+	if err != nil {
+		return api.PasswordRequirements{}, fmt.Errorf("could not get password settings: %w", err)
+	}
+
+	var requirements api.PasswordRequirements
+	requirements.LowercaseMin = settings.LowercaseMin
+	requirements.UppercaseMin = settings.UppercaseMin
+	requirements.NumberMin = settings.NumberMin
+	requirements.SymbolMin = settings.SymbolMin
+	requirements.LengthMin = settings.LengthMin
```

Comment:
Combine this with the GetSettings endpoint and build this in a single object, 
```
requirements := &api.Settings{
  PasswordRequirements: api.PasswordRequirements{
    LowercaseMin: settings.LowercaseMin,
    ...
  }
}
```

Reply:
done